staging/src/k8s.io/csi-translation-lib/plugins/azure_file_test.go

```diff
@@ -191,7 +191,8 @@ func TestTranslateAzureFileInTreePVToCSI(t *testing.T) {
 			name: "azure file volume",
 			volume: &corev1.PersistentVolume{
 				ObjectMeta: metav1.ObjectMeta{
-					Name: "file.csi.azure.com-sharename",
+					Name:        "file.csi.azure.com-sharename",
+					Annotations: map[string]string{resourceGroupAnnotation: "rg"},
```

Comment:
Is this annotation required on existing PVs? Is there a test case for a PV without the annotation?

Reply:
it's not required, added a new test case